geneiobio/geneiobio-iframe.xml

```diff
@@ -55,32 +55,35 @@
 	</configfiles>
 	<command>cat '$index' | tr -d '\n' > '$outfile'</command>
     <inputs>
-        <param name="proband_bam" type="data" format="bam,cram" label="Proband BAM file"/>
         <param name="proband_vcf" type="data" format="vcf,vcf_bgzip" label="Proband VCF file"/>
+        <param name="proband_bam" type="data" format="bam,cram" label="Proband BAM file" optional="true"/>
```

Comment:
Not sure if I can make input files optional using this method?

Reply:
https://docs.galaxyproject.org/en/latest/dev/schema.html#id44 yes